css-canvas-medical/styles.css

```diff
@@ -0,0 +1,234 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 900px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.off-white {
+  background-color: rgb(250 250 250);
+}
+
+.column-half {
+  width: 50%;
+}
+
+.blue-logo {
+  height: 18px;
+  width: 200px;
+  margin-left: 10px;
+}
+
+.contact-button {
+  padding: 10px 15px;
+  font-family: Avenir, sans-serif;
+  font-size: 10px;
+  color: rgb(255 255 255);
+  background-color: rgb(41 68 99);
+  border-radius: 2px;
+  border: 0;
+  margin-right: 30px;
+}
+
+.header-links {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.top-link {
+  font-family: Avenir, sans-serif;
+  color: rgb(111 111 111);
+  text-decoration: none;
+  font-size: 10px;
+}
+
+.border-bottom {
+  border-bottom: 0.5px solid rgb(220 220 220);
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.hero-row {
+  margin: 60px;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgb(0 0 0 / 30%), rgb(0 0 0 / 30%)), url("images/hero.jpg");
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-heading {
+  color: rgb(255 255 255);
+  font-family: "Tiempos Headline", sans-serif;
+  padding-left: 60px;
+  padding-right: 400px;
```

Comment:
Why is this necessary?

Reply:
is my padding bad practice? I the font color and style is there because the heading and paragraph have different fonts. Can you give me a little more direction with what you are trying to have me notice?

I am guessing there is a better way to do the alignment of the text?